tutorial/tutorial-gabc-01.html

```diff
@@ -89,7 +89,7 @@
 </pre>
 </div>
 
-<p>Here's the result after processing by gregorio and LaTeX:</p>
+<p>Here's the result after processing by gregorio and <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>:</p>
```

Comment:
Fine.  In that case, how come the examples that follow only highlight the note parts (between parentheses) in red and not all newly added text?